docs/release_notes/release_notes.md

```diff
@@ -9,52 +9,58 @@ has_children: true
 # Release Notes
 
 
-## Version 3.1.0 (March 2023)
+## Version 4.0.0 (May 2023)
 
-* ### Monitor Linux Groups & Users (Linux/Unix)
-    * Added savedsearch and dashboard panels to track the group and user changes for the Linux environment.
+* ### Notable Event Assignment
+    * User can assign a notable event to user and update the notable event status based the investigation from the Forensic dashboard.
+    * Added SOC Dashboard to get overview of notable events by the alert, assignee, status and severity.
+    * Updated Overview dashboard to filter the result based on the Status value.
 
-* ### Sophos Firewall Integration
-    * Mapped Sophos Firewall Traffic data with the Network_Traffic Model.
-    * Added new Sophos Firewall dashboard
-    ![alt]({{ site.baseurl }}/assets/sophos_firewall.png)
-    * To onboard the Sophos Firewall data, Refer [Sophos Firewall Data Onboarding]({{ site.baseurl }}/data_onboarding/network_devices/sophos_firewall/)
+![alt]({{ site.baseurl }}/assets/notable_event_assignment.png)
+
+* ### New Cyences Settings Page
+    * User can easily enable/disable the products as per their Splunk environment from the Product Setup page to enable/disable savedsearch and dashboards accordingly.
```

Comment:
* Users can use Cyences' new Settings page to customize the dashboards they want to see/hide and the alerts they enable/disable.
* The Settings page also allows you to see if the data-source configuration (index macro) is accurate and if your Splunk environment has data for it or not and modify the configuration if required.
* All the old configurations (Cyences default email configuration, macro setup, Sophos endpoint data collection creds, etc) are migrated to the new Cyences App Configuration Page.